addon/components/frost-button.js

```diff
@@ -1,14 +1,11 @@
+/**
+ * Component definition for the frost-button component
+ */
 import Ember from 'ember'
-const {
-  Component,
-  get,
-  isEmpty,
-  Logger,
-  typeOf,
-  ViewUtils
-} = Ember
+const {Component, Logger, ViewUtils, get, isEmpty, typeOf} = Ember
```

Comment:
1. It's easy to be consistent both way, you switch to one per line as soon as it's too large to fit 120 column (our defined max-line-length)

2. I've never had a case when I wanted to do that. I'd like to understand that use-case more. 

3. I disagree 100%, I find it physically painful to look at the multi-line with it would have fit on one line. Similar to something like:

```js
if ((foo === 'bar') && (bar === 'bad'}) {
  // do something
}
```
vs.
```js
if ((foo === 'bar') &&
    (bar === 'baz')) {
  // do something
}
```